src/components/pages/doc.js

```diff
@@ -97,6 +97,7 @@ function Doc({ initialState, PageTemplate }) {
   const [doc, setDoc] = useState(initialState.doc);
   const [comments, setComments] = useState([]);
   const [historyRevisions, setHistoryRevisions] = useState([]);
+  const [editedSectionKeys, setEditedSectionKeys] = useState([]);
```

Comment:
I left this in from an earlier commit. The edited state was the only section state (apart from internal plugin editors) that was not pulled up to the page level before. Now it is handled similar to the invalid and pending states of a section.